### PR TITLE
integration: add more docker-compose integration tests

### PR DESCRIPTION
**Pull request by nicks**
Hello @maiamcc,

Please review the following commits I made in branch nicks/dcimage3:

64df29015aa4f94113929304838132334c141b35 (2019-03-19 18:52:41 -0400)
integration: add more docker-compose integration tests

### CODE REVIEW

File: integration/dc_fixture_test.go [hunk 1]
```diff
@@ -44,6 +50,22 @@ func (f *dcFixture) dockerContainerID(name string) (string, error) {
...
+func (f *dcFixture) dockerCreatedAt(name string) string {
+	out := &bytes.Buffer{}
+	cmd := f.dockerCmd([]string{
+		"ps", "-q", "-f", fmt.Sprintf("name=%s", name), "--format", "{{.CreatedAt}}",
+	}, out)
+	err := cmd.Run()
+	if err != nil {
+		f.t.Fatal(fmt.Errorf("dockerCreatedAt: %v", err))
+	}
+	cID := strings.TrimSpace(out.String())
```
Comment (maiamcc) [inline integration/dc_fixture_test.go]: these variable names and error message 🤔

File: integration/dcfastbuild_test.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	f.CurlUntil(ctx, "dcfastbuild", "localhost:8000", "🍄 One-Up! 🍄")
+
+	f.ReplaceContents("cmd/dcfastbuild/main.go", "One-Up", "Two-Up")
+
+	ctx, cancel = context.WithTimeout(f.ctx, 30*time.Second)
+	defer cancel()
+	f.CurlUntil(ctx, "dcfastbuild", "localhost:8000", "🍄 Two-Up! 🍄")
+
+	createdAt2 := f.dockerCreatedAt("dcfastbuild")
+	assert.Equal(t, createdAt, createdAt2, "Container restarted changed in fast build")
```
Comment (maiamcc) [inline integration/dcfastbuild_test.go]: this message seems like maybe a typo

File: integration/onedc_test.go [hunk 3]
```diff
@@ -13,26 +13,26 @@ func TestOneDockerCompose(t *testing.T) {
...
 
-	f.dockerKillAll("onedc_web")
+	f.dockerKillAll("tilt")
 	f.TiltWatch("web")
 
 	ctx, cancel := context.WithTimeout(f.ctx, time.Minute)
 	defer cancel()
 
-	f.WaitUntil(ctx, "onedc_web up", func() (string, error) {
+	f.WaitUntil(ctx, "onedc up", func() (string, error) {
```
Comment (maiamcc) [inline integration/onedc_test.go]: this gets used a few times -- worth making a helper function for?

File: integration/dc_fixture_test.go (hunk 1 above)
Comment (nicks) [reply integration/dc_fixture_test.go]: these...are pretty idiomatically standard Go; can you elaborate more on why you don't like them?

File: integration/dcfastbuild_test.go (hunk 2 above)
Comment (nicks) [reply integration/dcfastbuild_test.go]: fixed

File: integration/onedc_test.go (hunk 3 above)
Comment (nicks) [reply integration/onedc_test.go]: good idea, fixed!

**Pull request merged**